-inputs.tf

```diff
@@ -14,9 +14,9 @@ variable "user_name" {
 }
 
 variable "user_home" {
-  description = "HOME path for transfer server user"
+  description = "HOME path for transfer server user. Mustn't start with /"
```

Comment:
Yes, but in this version there is no Terraform fixed version. If a customer or a third-party are using this module from the Terraform registry, it can break the compatibility. That's why I'll do it in a 2.0 version, with documentation about how to migrate between versions if needed.

Variables validation was added on Terraform 0.13. Do you think we should add it in this version? It has 2 years old
@aespejel-sg